.nais/kafka/behandlendeenhet.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  annotations:
+    dcat.data.nav.no/title: "Endring i brukers behandlende enhet"
```

Comment:
Kanskje bedre å bruke "arbeidstaker" enn "bruker"?

Reply:
Jepp, vi har gått bort fra å bruke ordet bruker om de sykmeldte. For så vidt gått bort fra å si sykmeldte også, og fokuserer heller på den viktigste rollen de har: arbeidstaker